recipe/install_ucx.sh

```diff
@@ -4,7 +4,7 @@ set -xeuo pipefail
 
 EXTRA_ARGS=""
 if [ "${cuda_compiler_version}" != "None" ]; then
-    EXTRA_ARGS="${EXTRA_ARGS} --with-cuda=${CUDA_HOME}"
+    EXTRA_ARGS="${EXTRA_ARGS} --with-cuda=${PREFIX}"
```

Comment:
Wonder if we should do something like this? Or perhaps drop the directory altogether (and just use `--with-cuda`)?

```suggestion
    EXTRA_ARGS="${EXTRA_ARGS} --with-cuda=`dirname $(which nvcc)`/.."
```

Reply:
Tried building locally, but none of the following worked:

- --with-cuda
- --with-cuda=${PREFIX}
- --with-cuda=`dirname $(which nvcc)`/..

Will continue looking for what may be happening.